test/UnitTests/TestFramework.UnitTests/Assertions/AssertTests.Contains.cs

```diff
@@ -362,6 +362,36 @@ public void Contains_ValueExpected_ItemDoesNotExist_ThrowsException()
         action.Should().Throw<AssertFailedException>().WithMessage("*20*");
     }
 
+    /// <summary>
+    /// Tests the Contains method (value overload) when the expected item is present.
+    /// </summary>
+    public void Contains_InNonGenericCollection_ValueExpected_ItemDoesNotExist_ThrowsException()
+    {
+        // Arrange
+        var collection = new ArrayList { 5, 10, "a" };
+
+        // Act
+        Action action = () => Assert.Contains(20, collection, "Item {0} not found", 20);
```

Comment:
Please also add `Assert.Contains(5, collection)` to ensure it's handled correctly and doesn't throw.